Documentation/poison-report-format.md

```diff
@@ -0,0 +1,63 @@
+# Interpreting the poison report
+
+The poison report looks something like this:
```

Comment:
Suggestion: cross reference the leak-detection document.

Reply:
Added, thanks.